src/message/renderMessages.js

```diff
@@ -10,30 +10,11 @@ import { isSameRecipient } from '../utils/message';
 import { isSameDay } from '../utils/date';
 
 type Props = {
-  auth: Auth,
-  actions: Actions,
-  subscriptions: any[],
-  users: Object[],
   messages: any[],
-  flags: Object,
   narrow: Narrow,
-  onLongPress: (message: Object) => void,
-  onHeaderLongPress: (item: Object) => void,
-  twentyFourHourTime: boolean,
 };
 
-export default ({
-  auth,
-  actions,
-  subscriptions,
-  users,
-  messages,
-  narrow,
-  onLongPress,
-  onHeaderLongPress,
-  flags,
-  twentyFourHourTime,
-}: Props) => {
+export default ({ messages, narrow }: Props) => {
```

Comment:
Great. That is one of the places where I thought it has to be refactored the most.